### PR TITLE
Set up application structure for User Client

### PR DESCRIPTION
**Pull request by SaraFrantzen**
[PT](https://www.pivotaltracker.com/story/show/175139259)
## This PR contains
- React application title: Good Morning News
- Converting App.js to App.jsx
- Cleaning

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (oliverochman) [inline src/index.css]: remove this file

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```
Comment (oliverochman) [inline public/manifest.json]: remove the "icons" key here

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 1 above)
Comment (SaraFrantzen) [reply src/index.css]: done

File: public/manifest.json (hunk 2 above)
Comment (SaraFrantzen) [reply public/manifest.json]: done

**Pull request merged**